tools/run_tests/performance/scenario_config.py

```diff
@@ -215,6 +215,29 @@ def scenarios(self):
           categories=smoketest_categories+[SCALABLE])
 
       yield _ping_pong_scenario(
+          'cpp_generic_async_streaming_qps_1channel_1MBmsg_%s' % secstr,
```

Comment:
nit: I think other perf tests use "secstr_msgsize" naming pattern and not  "msgsize_secstr". Also, they're not using the "msg" suffix.